templates/html-header.twig

```diff
@@ -30,7 +30,7 @@
     {% endif %}
 
     {% if canonical_link %}
-        <link rel="canonical" href="{{ canonical_link }}">
+        <link rel="canonical" href="{{ canonical_link ~ '/' }}">
```

Comment:
Are we sure the trailing slash is always missing? Could it cause issues if we add one where there's already one? 🤔

Reply:
All the pages I checked with self assigned URLs, did not have the trailing slash, but I will see if I can add a check for the trailing slash before adding it.